Delete stale provider dashboard (pattern fly) charts code.

kavyanekkalapu opened the pull request:
Provider dashboard charts converted to carbon except heatmap chart in https://github.com/ManageIQ/manageiq-ui-classic/pull/7658, in this pr, i am removing all that old code which is useless now.
@miq-bot assign @Fryguy 
@miq-bot add-reviewer @Fryguy

Fryguy commented:
@jrafanie I want to give you the honors 😆

jrafanie commented:
> Provider dashboard charts converted to carbon except heatmap chart in #7658, in this pr, i am removing all that old code which is useless now.

`+1 −1,640 `

✂️ 🗑️ 🚽  Very nice.  

I don't see much conditional logic that controlled which style was used: patternfly or carbon.  Did that conditional logic exist and can it be removed also?

I don't know if that one line added is the single conditional or if there's more deletions out there...

kavyanekkalapu commented:
> > Provider dashboard charts converted to carbon except heatmap chart in #7658, in this pr, i am removing all that old code which is useless now.
> 
> `+1 −1,640 `
> 
> ✂️ 🗑️ 🚽 Very nice.
> 
> I don't see much conditional logic that controlled which style was used: patternfly or carbon. Did that conditional logic exist and can it be removed also?
> 
> I don't know if that one line added is the single conditional or if there's more deletions out there...

@jrafanie I couldn't understand the comment fully, here, patternfly charts code are angular directives, i deleted all of the js files and html files related to those directives and  deleted references of files in our provider dashboard charts haml files too.

jrafanie commented:
> @jrafanie I couldn't understand the comment fully, here, patternfly charts code are angular directives, i deleted all of the js files and html files related to those directives and deleted references of files in our provider dashboard charts haml files too.

Sorry about that.  I guess I don't understand how it chose to use patternfly or carbon so assumed some code decided which would be used.  I'm good with merging it as is but was unsure if there was code that made decisions about which to use.

kavyanekkalapu commented:
> > @jrafanie I couldn't understand the comment fully, here, patternfly charts code are angular directives, i deleted all of the js files and html files related to those directives and deleted references of files in our provider dashboard charts haml files too.
> 
> Sorry about that. I guess I don't understand how it chose to use patternfly or carbon so assumed some code decided which would be used. I'm good with merging it as is but was unsure if there was code that made decisions about which to use.
Ah ok, there is no conditional logic, in haml files, earlier we used to refer PF charts , we replaced them with carbon charts components in https://github.com/ManageIQ/manageiq-ui-classic/pull/7658, so we are not using the PF charts anymore, hence, i deleted all that code.

jrafanie commented:
> Ah ok, there is no conditional logic, in haml files, earlier we used to refer PF charts , we replaced them with carbon charts components in #7658, so we are not using the PF charts anymore, hence, i deleted all that code.

Thanks, I feel better knowing that all potentially deletable code has been considered for the 🗑️ ✂️ 🚽 . 

✂️ 🔥 ✂️ 🔥 ✂️ 🔥 

🍰  🍪 👏 🙇 😍 🎉

jrafanie commented:
Let's do this.
